cmd/src/actions_exec.go

```diff
@@ -216,8 +216,8 @@ Format of the action JSON files:
 			return errors.Wrap(err, "Validation of action failed")
 		}
 
-		// Build Docker images etc.
-		err = prepareAction(ctx, action)
+		// Build Docker images, pull Docker images from registries, etc.
```

Comment:
We don't build images anymore :)

```suggestion
		// Pull Docker images from registries, get image digests, etc.
```